rpms/s3/s3rpm.spec

```diff
@@ -17,6 +17,16 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+# Following diagram represent the scriptlets flags for the RPM install/upgrade/uninstall phases in spec files.
+# new -> new RPM 
+# old -> old RPM
+#--------------------------------------------------------------------------------------------------------------------------------------
+#                Install RPM                                  Upgrade RPM                                   Uninstall RPM
+#                    |                                             |                                              |
+#                    |                          pre $1 == 2 (new)  | preun $1 == 1 (old)                          |
+#   pre $1 = 1 (new) | post $1 == 1 (new)       post $1 == 2 (new) | postun $1 == 1 (old)     preun $1 == 0 (new) | postun $1 == 0(new)
+#--------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
pre $1 = 1 (new) | post $1 == 1 (new)       post $1 == 2 (new) | postun $1 == 1 (old)     preun $1 == 0 (current) | postun $1 == 0(current)

Reply:
not clear.